src/components/Compose.js

```diff
@@ -46,12 +47,16 @@ function Compose(props) {
   const input = useRef();
 
   useEffect(() => {
-    const onKeyDown = (e) => {
+    const onKeyDown = e => {
```

Comment:
Nitpicking: unnecessary change.
```suggestion
    const onKeyDown = (e) => {
```